zerver/openapi/zulip.yaml

```diff
@@ -21061,7 +21061,8 @@ paths:
       summary: Update a user group
       tags: ["users"]
       description: |
-        Update the name or description of a [user group](/help/user-groups).
+        Update the name, description or any of the permission settings
+        of a [user group](/help/user-groups).
```

Comment:
Have not added "Changes" entries here since we already have that for individual settings.